angularjs/angular.d.ts

```diff
@@ -453,7 +453,7 @@ declare module angular {
          * following compilation. The observer is then invoked whenever the
          * interpolated value changes.
          */
-        $observe(name: string, fn: (value?: any) => any): Function;
+        $observe<T>(name: string, fn: (value?: T) => any): Function;
```

Comment:
The point is to avoid implicit "any" by omission, especially together with "noImplicitAny". If you have to rely on the developer explicitly adding a type annotation, that defeats the purpose. Please also see the referenced issue in the commit message and the example.